pkg/utils/sysfs.go

```diff
@@ -137,3 +153,53 @@ func readFileInt(path string) (int, error) {
 	i, err := strconv.ParseInt(strings.TrimSpace(string(data)), 10, 32)
 	return int(i), err
 }
+
+func writeFileStr(path string, value string) error {
+	backupPath := path + ".original"
+
+	// Take a backup the original file
+	err := copyFile(path, backupPath)
```

Comment:
Sorry, I don't get this backup business at all :monocle_face: 

Do you really expect to be able to create some backup files in sysfs like `/sys/devices/system/cpu/cpufreq/policy1/scaling_governor.original`?

Doesn't kernel reject invalid/unknown governors?

Reply:
My guess is that this is an attempt to address one of my earlier review comment about atomic updates to multiple CPU cores' governor. That would require reading and saving the earlier state per CPU then trying to roll back changes if the update fails halfway through. But that cannot be done like this.

Reply:
Thanks for the review. I have reverted those extra lines of dealing with backup files. PTAL